addons/comment-scroll/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Scrollable comments",
+  "description": "Makes comments scrollable, which can reduce the height of the page.",
+  "credits": [
+    {
+      "name": "Genius398",
+      "link": "https://github.com/Genius398/"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": [
+        "https://scratch.mit.edu/users/*",
+        "https://scratch.mit.edu/projects/*",
+        "https://scratch.mit.edu/studios/*"
+      ]
+    }
+  ],
```

Comment:
You should rewrite the CSS differently for scratch-www and scratchr2. Also, add yourself into the credits section as you PR'd it